cn/ios/objects.mdown

```diff
@@ -2,17 +2,17 @@
 
 ## PFObject
 
-Parse 上的数据储存建立在 `%{ParseObject}` 的基础上。每个 `%{ParseObject}` 包含 JSON 兼容数据的键值对。该数据没有计划性，即您不需要事先指定每个 `%{ParseObject}` 上存在的键。您只需随意设置您需要的键值对，我们的后台会储存它们。
+Parse 上的数据储存建立在 `ParseObject` 的基础上。每个 `ParseObject` 包含 JSON 兼容数据的键值对。该数据是无需预定义结构的，即您不需要事先指定每个 `ParseObject` 上存在的键。您只需随意设置您需要的键值对，我们的后台会储存它们。
```

Comment:
I modify this document base on the en/ios/objects.mdown, it doesn't has %{}.Now, I think I should modify the en/ios/objects.mdown instead of cn/ios/objects.mdown